sopel/db.py

```diff
@@ -93,14 +93,19 @@ class PluginValues(BASE):
 
 
 class SopelDB(object):
-    """*Availability: 5.0+*
+    """Database object class.
 
-    This defines an interface for basic, common operations on a sqlite
-    database. It simplifies those common operations, and allows direct access
-    to the database, wherever the user has configured it to be.
+    :param config: Sopel's configuration settings
+    :type config: :class:`sopel.config.Config`
 
-    When configured with a relative filename, it is assumed to be in the directory
-    set (or defaulted to) in the core setting ``homedir``.
+    This defines a simplified interface for basic, common operations on the
+    bot's database. Direct access to the database is also available, to serve
+    more complex plugins' needs.
+
+    When configured to use SQLite with a relative filename, the file is assumed
+    to be in the directory named by the core setting ``homedir``.
+
+    .. versionadded:: 5.0
```

Comment:
I'd add a `.. versionchanged:: 7.0` to talk about the switch to SQLAlchemy here.

Reply:
Done, see https://github.com/sopel-irc/sopel/compare/deea18348561879d1fd03e6ec00ac67f1e924327..75d276652e2288f28b7fe6f9805b43c6a7be2dab